setup.cfg

```diff
@@ -20,8 +20,9 @@ classifiers =
     Operating System :: POSIX :: Linux
     Operating System :: MacOS :: MacOS X
     Programming Language :: Rust
-    Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
```

Comment:
Does that mean sourmash is installable on a py3.10 env but sourmash itself does not support >py3.8 in the dev code?

Reply:
the tags are inclusive, so it means it is being advertised as supporting 3.8, 3.9, and 3.10.